app/_kong_plugins/app-dynamics/index.md

```diff
@@ -29,11 +29,121 @@ topologies:
 icon: app-dynamics.png
 
 categories:
-  - analytics-monitoring
+  - analytics
+  - monitoring
```

Comment:
```suggestion
  - analytics-monitoring
  
tags:
  - analytics
  - monitoring
```
The category expects `analytics-monitoring` to sort the plugins in the right place in the plugin hub: https://deploy-preview-674--kongdeveloper.netlify.app/plugins/?category=analytics-monitoring

You can add these add individual tags though.